env.example

```diff
@@ -17,4 +17,11 @@ export MOCK_MSI_CERT="replace_with_value_output_by_hack/devtools/msi.sh"
 export MOCK_MSI_TENANT_ID="replace_with_value_output_by_hack/devtools/msi.sh"
 export PLATFORM_WORKLOAD_IDENTITY_ROLE_SETS="replace_with_value_output_by_hack/devtools/msi.sh"
 
+# you will need this to run-rp , vpn and ci-rp using Docker compose
+export REGISTRY=registry.access.redhat.com
+export LOCAL_ARO_RP_IMAGE=localhost/aro
+export VERSION=latest
+export TAG=latest
+export LOCAL_VPN_IMAGE=arosvcdev.azurecr.io/vpn
```

Comment:
nit: `docker compose` for me locally attempts to optimize here, and pull the image before building. This results in a warning that auth to the ACR failed. Definitely not a blocker, but I wonder of making this `LOCAL_VPN_IMAGE=vpn` and the `LOCAL_ARO_RP_IMAGE=aro` for simplicity makes sense or not?

Reply:
Updated the env file with  variables `LOCAL_VPN_IMAGE=vpn `and the `LOCAL_ARO_RP_IMAGE=aro `for simplicity